5.0/5.0.7/debian/Dockerfile

```diff
@@ -9,11 +9,12 @@ ENV PLONE_MAJOR=5.0
 ENV PLONE_VERSION=5.0.7
 ENV PLONE_MD5=347e787d10a6a02d8156ed8c05effcdb
 
-LABEL version=$PLONE_VERSION \
+LABEL plone=$PLONE_VERSION \
 	  os="debian" \
 	  os.version="8" \
-	  name="plone"
-
+	  name="Plone 5 " \
```

Comment:
Where ? :)
I checked in two different editors with linting for Dockerfiles and I can't see a extra space :)
![screen_subl](https://cloud.githubusercontent.com/assets/358860/24755958/5bf00786-1adc-11e7-8fba-7be97f07288a.png)